packages/typescript-configs/web-application.json

```diff
@@ -2,7 +2,6 @@
     "$schema": "https://json.schemastore.org/tsconfig",
     "extends": "./react.json",
     "compilerOptions": {
-        "allowImportingTsExtensions": true,
         "moduleResolution": "bundler"
```

Comment:
I there a specific reason why we haven't add the default `exclude` to this config?

Reply:
I'm not sure what we would exclude here in a web application. There is no dist folders